cli/lib/command/internal_command_doctor.dart

```diff
@@ -135,5 +135,70 @@ class InternalCommandDoctor extends InternalCommand {
         );
       }
     }
+
+    // Check gh CLI installation and authentication
+    {
+      bool ghExists;
+      try {
+        ghExists = context
+            .withSilence(true)
+            .command(['gh', '--version'])
+            .announce('Checking if GitHub CLI is installed.')
+            .runSync()
+            .isSuccess();
+      } catch (e) {
+        ghExists = false;
+      }
+
+      context.printToConsole(
+        '''[${boolToCheckmark(ghExists)}] gh --version # ${ghExists ? "installed" : "not found"}''',
+      );
+
+      if (!ghExists) {
+        context.printToConsole('''    X Install GitHub CLI using:''');
+        context.printToConsole(
+          '''      gh not installed''',
```

Comment:
Let's give people nice install instructions
```suggestion
          '''      https://github.com/cli/cli#installation''',
```